crates/core/app.rs

```diff
@@ -1971,6 +1975,9 @@ fn flag_no_ignore_dot(args: &mut Vec<RGArg>) {
         "\
 Don't respect .ignore files.
 
+This does **not** affect whether ripgrep will ignore files and directories
```

Comment:
This is technically inaccurate in that a .ignore file with negation patterns could affect whether ripgrep ignores dotfiles, but it felt overly verbose to clarify that as part of the note. If you have a better wording proposal that is more accurate, I'd be happy to update accordingly